packages/common-integration-tests/src/storage/storage.ts

```diff
@@ -71,169 +70,26 @@ export function runStorageServiceTests(
-          testingStoreName,
-          key
-        );
-        expectWithMessage(() => {
-          expect(getStringResponse.type).toEqual(StorageGetResponse.Found);
-        }, `expected Found, received ${getStringResponse.toString()}`);
-        expect(getStringResponse.value()?.string()).toEqual(stringValue);
-
-        // put/get a bytes value
-        const bytesValue = new Uint8Array([1, 2, 3, 4]);
-        const putBytesResponse = await storageClient.putBytes(
-          testingStoreName,
-          key,
-          bytesValue
-        );
-        switch (putBytesResponse.type) {
-          case StoragePutResponse.Success: {
+      // WithStore creates a store then deletes it
+      await WithStore(storageClient, storeName, async () => {
+        const alreadyExistResponse = await storageClient.createStore(storeName);
+        switch (alreadyExistResponse.type) {
+          case CreateStoreResponse.AlreadyExists: {
             break;
           }
-          case StoragePutResponse.Error: {
+          case CreateStoreResponse.Error: {
             throw new Error(
-              `failed to put key: ${putBytesResponse.message()} ${putBytesResponse.toString()}`
+              `failed to create store, expected AlreadyExists response, error: ${alreadyExistResponse.message()} exception: ${alreadyExistResponse.toString()}`
             );
           }
-        }
-        const getBytesResponse = await storageClient.get(testingStoreName, key);
-        expectWithMessage(() => {
-          expect(getBytesResponse.type).toEqual(StorageGetResponse.Found);
-        }, `expected Found, received ${getBytesResponse.toString()}`);
-        expect(getBytesResponse.value()?.bytes()).toEqual(bytesValue);
-
-        const deleteResponse = await storageClient.delete(
-          testingStoreName,
-          key
-        );
-        switch (deleteResponse.type) {
-          case StorageDeleteResponse.Success: {
-            break;
-          }
-          case StorageDeleteResponse.Error: {
+          case CreateStoreResponse.Success: {
             throw new Error(
-              `failed to delete key in store: ${deleteResponse.message()} ${deleteResponse.toString()}`
+              'store already exists, we should not be able to create it again'
             );
           }
         }
       });
     });
-    it('should return an undefined value for a key that doesnt exist', async () => {
-      await WithStore(storageClient, testingStoreName, async () => {
-        const key = v4();
-        const getResponse = await storageClient.get(testingStoreName, key);
-        expectWithMessage(() => {
-          expect(getResponse.type).toEqual(StorageGetResponse.NotFound);
-        }, `expected NotFound, received ${getResponse.toString()}`);
-        expect(getResponse.value()).toBeUndefined();
-      });
-    });
     it('should return store not found error for deleting a store that doesnt exist', async () => {
```

Comment:
moved 'should return store not found error for deleting a store that doesnt exist' to the storage control plane block of tests